docs/mobile/development.md

```diff
@@ -79,12 +79,12 @@ mobile-teach preview /path/to/lesson-file.yml
 ```
 
```

Comment:
Note for the text before: should we mention what NPM is and how to install it?

Reply:
I currently link to the download and installation page, which should hopefully cover installation, but yes I can add a quick explanation of what npm is.